.github/workflows/publish-swift.yml

```diff
@@ -0,0 +1,140 @@
+          cp ../target/aarch64-apple-ios/release/libbreez_sdk_bindings.a bindings-swift/breez_sdkFFI.xcframework/ios-arm64/breez_sdkFFI.framework/breez_sdkFFI
+          cp ../target/ios-universal-sim/release/libbreez_sdk_bindings.a bindings-swift/breez_sdkFFI.xcframework/ios-arm64_x86_64-simulator/breez_sdkFFI.framework/breez_sdkFFI
+          cp ../target/darwin-universal/release/libbreez_sdk_bindings.a bindings-swift/breez_sdkFFI.xcframework/macos-arm64_x86_64/breez_sdkFFI.framework/breez_sdkFFI
+          rm bindings-swift/Sources/BreezSDK/breez_sdkFFI.h
+          rm bindings-swift/Sources/BreezSDK/breez_sdkFFI.modulemap
+
+      - name: Compress xcframework
+        working-directory: breez-sdk/libs/sdk-bindings/bindings-swift
+        run: |
+          zip -9 -r breez_sdkFFI.xcframework.zip breez_sdkFFI.xcframework
+          echo "XCF_CHECKSUM=`swift package compute-checksum breez_sdkFFI.xcframework.zip`" >> $GITHUB_ENV
+
+      - name: Archive xcframework
+        uses: actions/upload-artifact@v4
+        with:
+          name: breez_sdkFFI-${{ inputs.package-version || intouts.ref }}.xcframework
+          path: breez-sdk/libs/sdk-bindings/bindings-swift/breez_sdkFFI.xcframework
+
+      - name: Update swift package
+        working-directory: breez-sdk/libs/sdk-bindings/bindings-swift
+        run: |
+          # Update package definition
+          sed 's#.binaryTarget(name: "breez_sdkFFI", path: "./breez_sdkFFI.xcframework"),#.binaryTarget(name: "breez_sdkFFI", url: "https://github.com/breez/breez-sdk-swift/releases/download/${{ inputs.package-version }}/breez_sdkFFI.xcframework.zip", checksum: "${{ env.XCF_CHECKSUM }}"),#;/.testTarget(name: "BreezSDKTests", dependencies: \["BreezSDK"\]),/d' Package.swift > ../../../../breez-sdk-swift/Package.swift
+          # Update language bindings
+          cp -r Sources ../../../../breez-sdk-swift
+
+      - name: Update cocoapods definitions
+        working-directory: breez-sdk-swift
+        run: |
+          sed -i '' 's#^.\{2\}spec.version.*$#  spec.version                = "${{ inputs.package-version }}"#' breez_sdkFFI.podspec
+          sed -i '' 's#^.\{2\}spec.version.*$#  spec.version                = "${{ inputs.package-version }}"#' BreezSDK.podspec
+
+      - name: Tag swift package
+        working-directory: breez-sdk-swift
+        if: ${{ inputs.publish }}
+        run: |
+          git add Package.swift
+          git add Sources
+          git add breez_sdkFFI.podspec
+          git add BreezSDK.podspec
+          git commit -m "Update Breez SDK Swift bindings to version ${{ inputs.package-version }}"
+          git push
+          git tag ${{ inputs.package-version }} -m "${{ inputs.package-version }}"
+          git push --tags
+
+      - name: Release and attach XCFramework binary artifact
+        if: ${{ inputs.publish }}
+        uses: ncipollo/release-action@v1
```

Comment:
I seem to remember this action had issues in the [breez-sdk-liquid](https://github.com/breez/breez-sdk-liquid) repo when using a token. There we use `softprops/action-gh-release@v2` as the release action instead.

Reply:
Good insight thanks! Will use `softprops/action-gh-release@v2` here too. Good for consistency too! 👏